test/algorithms/clear_multi.cpp

```diff
@@ -42,12 +56,12 @@ void test_all()
     typedef bg::model::multi_linestring<ls> mls;
     typedef bg::model::multi_polygon<poly> mpoly;
 
-    test_geometry<Point>("POINT(0 0)", 1);
-    test_geometry<ls>("LINESTRING(0 0,0 1)", 0);
-    test_geometry<poly>("POLYGON((0 0,0 1,1 0,0 0))", 0);
-    test_geometry<mpoint>("MULTIPOINT((0 0),(0 1),(1 0),(0 0))", 0);
-    test_geometry<mls>("MULTILINESTRING((0 0,0 1),(1 0,0 0))", 0);
-    test_geometry<mpoly>("MULTIPOLYGON(((0 0,0 1,1 0,0 0)),((10 0,10 1,11 0,10 0)))", 0);
+    test_geometry<Point>("POINT(0 0)", 1, 1);
```

Comment:
why not `0` after clearing?

Reply:
Because Boost.Geometry Point can't be empty.